components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -432,7 +432,7 @@ func k8sClientProvider(kcfg string) (client.Client, error) {
 		return nil, err
 	}
 
-	sch := runtime2.NewScheme()
+	sch := scheme.Scheme
```

Comment:
the BTP-Operator cleanup needs to list `Namespaces` because `ServiceInstances` and `ServiceBindings` are namespaced resources and we need to clean them up in each namespace. As a result, I think it's best to use the default scheme from client-go rather than form a scheme from scratch and register core resources one by one when needed.